src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Todo } from '../../types/Todo';
+
+type TodoHeaderProps = {
+  todos: Todo[];
+  isAdding: boolean;
+  handleToggleAll: () => void;
+  handleAddTodo: (event: React.FormEvent<HTMLFormElement>) => void;
```

Comment:
```suggestion
  onToggleAll: () => void;
  onAddTodo: (todo: string) => void;
```

1. Maintain consistent naming
2. Dont expose the internals of the component